packages/jobs/lib/env.ts

```diff
@@ -1,3 +1,3 @@
 import { ENVS, parseEnvs } from '@nangohq/utils';
 
-export const envs = parseEnvs(ENVS.required({ ORCHESTRATOR_SERVICE_URL: true }));
+export const envs = parseEnvs(ENVS.required({ ORCHESTRATOR_SERVICE_URL: true, NANGO_REDIS_URL: true }));
```

Comment:
this is making Redis mandatory for enterprise setup no? are we ok with that? is it gonna break existing enterprise customer? cc @khaliqgant

Reply:
the locking would still works with an in-memory kvstore if there is a single instance? no?

Reply:
Evertrue is not using redis fwiw